lib/util.js

```diff
@@ -57,9 +57,9 @@ exports.formatTimestampToDate = function(timestamp) {
 };
 
 exports.formatMs = function(ms) {
-  const seconds = Math.floor((ms / 1000) % 60);
-  const minutes = Math.floor((ms / (1000 * 60)) % 60);
-  const hours = Math.floor((ms / (1000 * 60 * 24)) % 60);
+  const seconds = Math.floor(ms / 1000) % 60;
+  const minutes = Math.floor(ms / (1000 * 60)) % 60;
+  const hours = Math.floor(ms / (1000 * 60 * 60));
   let formattedTime = '';
 
```

Comment:
Could you replace constructing `formattedTime` logic with the following changes?

```js
if (hours) {
  formattedTime += hours.toString(10).padStart(2, '0') + ' hrs';

  (minutes || seconds) && (formattedTime += ', ');
}

if (minutes) {
  formattedTime += minutes.toString(10).padStart(2, '0') + ' min';

  seconds && (formattedTime += ', ');
}

if (seconds) formattedTime += seconds.toString(10).padStart(2, '0') + ' sec';
```

Reply:
Hi, @m-sureshraj! :D.

Of course I can! :D. But now, checking it, I think it would like better to change the structure and use an array.
I mean this (give me your opinion):

```
const formattedTime = [];

  if (hours) formattedTime.push(hours.toString(10).padStart(2, '0')) + ' hrs';
  if (minutes) formattedTime.push(minutes.toString(10).padStart(2, '0')) + ' min';
  if (seconds) formattedTime.push(seconds.toString(10).padStart(2, '0')) + ' sec';

  return formattedTime.join(', ');
```

It would get the same result you want (I didn't get previously that there would be a problem when, as example, there weren't any seconds left, in the formatting), and it's more compact and readable. And you dont need to use that extra short-circuit conditions this way. Do you like it?

If not and if you want it the way you explicitly asked for, I have no problem adding that instead. What do you say then?

Reply:
Neat! I like your solution :+1:.

Reply:
Change pushed :+1: .